app/Exceptions/Handler.php

```diff
@@ -29,22 +31,27 @@ class Handler extends ExceptionHandler
     /**
      * Report or log an exception.
      *
-     * @param  \Exception  $exception
+     * @param Throwable $exception
      * @return void
+     *
+     * @throws Exception
      */
-    public function report(Exception $exception)
+    public function report(Throwable $exception)
     {
+        reportException($exception);
```

Comment:
Sorry just caught this, should the parent method be called first?

Reply:
I believe in most bug-catching frameworks, their logic usually goes first? Ref: https://docs.sentry.io/platforms/php/laravel/ and https://github.com/VATSIM-UK/core/blob/master/app/Exceptions/Handler.php#L57